src/crewai/agents/agent_builder/utilities/yes_no_cli_validator.py

```diff
@@ -0,0 +1,8 @@
+from prompt_toolkit.validation import Validator, ValidationError
+
+
+class YesNoValidator(Validator):
+    def validate(self, document):
+        text = document.text.lower()
+        if text not in ["y", "n", "yes", "no"]:
+            raise ValidationError(message="Please enter Y/N")
```

Comment:
_Question_: Again, this could've been a simple validator function

```suggestion
def validate_user_input(user_input: str):
    if document.text.lower() not in ["y", "n", "yes", "no"]:
         raise Exception("Error with the input")
```